### PR TITLE
Adds rake task to upload mezzanine files to existing objects; address…

### PR DESCRIPTION
**Pull request by coblej**
…es DDR-452.

### CODE REVIEW

**Comment by coblej** [comment]
Scenario is a folder of mezzanine files for existing repo objects.  Files have same names as master files but with different extensions (.jpg instead of .tif), so file name without extension should match local_id of Component to which mezzanine is to be added.

Easiest path to follow through what the code is doing ...
rake task dul_hydra:batch:mezzanine
... calls DulHydra::Batch::Scripts::AddMezzanineFiles
... queues up AddMezzanineFileJob for each mezzanine file in folder
... calls (service) AddMezzanineFile

File: app/jobs/add_mezzanine_file_job.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AddMezzanineFileJob
+  extend AbstractJob
+
+  @queue = :batch
+
+  def self.perform(user:, filepath:, mezzanine_file:)
+    AddMezzanineFile.new(args).process
```
Comment (dchandekstark) [inline app/jobs/add_mezzanine_file_job.rb]: Does this work, above?

**Comment by coblej** [comment]
@dchandekstark Hold off on merging this.  When I get a chance, I'm going to update the PR by replacing 'mezzanine' with 'intermediate'.

**Comment by coblej** [comment]
@dchandekstark PR has been updated and is now ready for your review / merge.

**Pull request merged**